recipes/vulkan-memory-allocator/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class VulkanMemoryAllocatorConan(ConanFile):
+    name = "vulkan-memory-allocator"
+    license = "MIT"
+    homepage = "https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Easy to integrate Vulkan memory allocation library."
+    topics = ("vulkan", "memory-allocator", "graphics")
+    requires = ("vulkan-headers/[>=1.0]")
```

Comment:
the recipes at cci don't use version ranges.


```suggestion
    requires = ("vulkan-headers/1.2.154.0")
```

Reply:
I would also advice to never use `requires`, but list dependencies in `requirements()` instead. It makes CCI recipes more uniform (several recipes have conditional requirements, so these requirements can only live in `requirements()`).

Reply:
I thought it is a good idea to use a version range because the library does not require a specific version of Vulkan-Headers but detects what is available.

But OK, I think there is no other option. Thank you.